rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -137,7 +138,15 @@ void processMinersFees() throws IOException, BlockStoreException {
         // the update of bytes is needed, because BridgeSupport creation could alter
         // the storage when getChainHead is null (specially in production)
         processingRepository = processingRepository.startTracking();
-        RemascFederationProvider federationProvider = new RemascFederationProvider(processingRepository);
+        BridgeSupport bridgeSupport = new BridgeSupport(
+                repository,
```

Comment:
I think this should be the `processingRepository` as @ajlopezrsk suggested offline